md/release-notes.md

```diff
@@ -364,6 +364,22 @@ Please use the Tomcat bundle instead, or see the [Custom Deployment](deploy-bund
 ## Bug fixes
 ### Fixes in Documentation
 * BS-19323 Bonita documentation is not clear enough about PostgreSQL versions supported
+* BST-151 Default outgoing transition not executed when  there is more than one outgoing transition from inclusive gateway
+* BST-269 Document workaround for macos - ssh issue
+* BST-171 Can't share a project on github using HTTPS and double authentication factor
+* BST-341 On Mac, Studio does not start if recent Java versions installed
+
+### Fixes in Bonita 7.9.2 (2019-08-01)
+#### Fixes in Engine component
+* BS-19299 GET REST API connector raised an exception if answer HTTP code is 204 or 404
```

Comment:
The fix has been done in the connector not in the Engine

Reply:
We usually do not have such a component to report bug fixing. Either Studio or Engine.

Reply:
And the exact description would be `GET REST API connector raised an exception if answer body is null` (which is often the case when the code is 204)

Reply:
Does it still make sense to separate Portal, Engine, Connectors, ...?
Can't we just say Runtime env (We + Portal + Engine) / Development env (Studio + UID) ?
That sounds more meaningful to my understanding of what a Bonita Customer might expect...

Reply:
This question will be addressed for 7.10. I'm taking notes.